CONTRIBUTING.md

```diff
@@ -1,36 +1,31 @@
-## How to Contribute to Vulkan Source Repositories
+## How to Contribute to the VulkanTools Repository
 
 ### **The Repository**
 
-The source code for The Vulkan-ValidationLayer components is sponsored by Khronos and LunarG.
-* [Khronos Vulkan-ValidationLayers](https://github.com/KhronosGroup/Vulkan-ValidationLayers)
+The source code for the VulkanTools components is sponsored by Khronos and LunarG.
+* [VulkanTools](https://github.com/LunarG/VulkanTools)
 
 
 ### **The Vulkan Ecosystem Needs Your Help**
 
-The Vulkan validation layers are one of the larger and more important components in this repository.
-While there are often active and organized development efforts underway to improve their coverage,
-there are always opportunities for anyone to help by contributing additional validation layer checks
-and tests for these validation checks.
+The tools in VulkanTools are important components in the Vulkan Ecosystem.  While there are often
+active and organized development efforts underway to improve these tools, there are always
+opportunities for anyone to help by contributing new tools or additional features to existing tools.
+these validation checks.
```

Comment:
I think this sentence was supposed to be deleted, as there is a period on the previous line...

Reply:
> ### **The Vulkan Ecosystem Needs Your Help**
 I guess I would like the title to be more like: Contributions welcomed. The Vulkan ecosystem will benefit